docker-compose-postgres.yml

```diff
@@ -37,8 +37,6 @@ services:
       - 6379:6379
     expose:
       - 6379
-    networks:
```

Comment:
I already corrected this in a recent PR (by adding the network below, rather than removing this one here). You can leave this file untouched